docs/design/kata-vra.md

```diff
@@ -43,7 +43,7 @@ and perform DMA transactions _anywhere_.
 
 The second feature is ACS (Access Control Services), which controls which
 devices are allowed to communicate with one another and thus avoids improper
-routing of packets irrespectively of whether IOMMU is enabled or not.
+routing of packets `irrespectively` of whether IOMMU is enabled or not.
```

Comment:
@GabyCT is this because of the spell check?

Reply:
@cmaf exactly kata spell check sends those warnings which makes `make static checks` fail, before on the tests repository we had a file to add words and so on but as that repo soon will be disabled we need to do this